xtraplatform-features/src/main/java/de/ii/xtraplatform/features/app/FeatureProviderDataMigrationV1V2.java

```diff
@@ -40,19 +45,97 @@ public FeatureProviderDataV2 migrate(FeatureProviderDataV1 entityData) {
                 .featureProviderType(entityData.getFeatureProviderType())
                 .nativeCrs(entityData.getNativeCrs());
 
-        //TODO: transform flat properties structure from entityData into new nested structure
 
         entityData.getTypes().values().forEach(featureType -> {
 
+            String featureTypePath = "/" + featureType.getName();
             ImmutableFeatureTypeV2.Builder featureTypeNew = new ImmutableFeatureTypeV2.Builder()
-                    .path("");
+                    .path(featureTypePath);
```

Comment:
I think this works for the given test case, but not in general. The feature type name does not always equal the main table name. So it would be better to use the first path element of the first property.